css/properties/aspect-ratio.json

```diff
@@ -53,92 +46,23 @@
               "version_added": false
             },
             "safari": {
-              "version_added": "14",
-              "partial_implementation": true,
-              "notes": "Safari 14 adds internal support only for mapped values"
+              "version_added": false
             },
             "safari_ios": {
-              "version_added": "14",
-              "partial_implementation": true,
-              "notes": "Safari 14 adds internal support only for mapped values"
+              "version_added": false
             },
             "samsunginternet_android": {
-              "version_added": "12.0",
-              "partial_implementation": true,
-              "notes": "Samsung Internet 12.0 adds internal support only for mapped values"
+              "version_added": false
             },
             "webview_android": {
-              "version_added": "79",
-              "partial_implementation": true,
-              "notes": "Webview 79 adds internal support only for mapped values"
+              "version_added": false
             }
           },
           "status": {
-            "experimental": true,
+            "experimental": false,
             "standard_track": true,
             "deprecated": false
           }
-        },
```

Comment:
Sorry, Rachel. I should've been more clear about what I was asking for. I'd like you to undo the changes from this line down. In other words, leave the `internal-value` feature in the file (which will be removed in my PR), but keep the changes from above this line.